jwst/mrs_imatch/mrs_imatch_step.py

```diff
@@ -163,7 +229,11 @@ def _apply_sky_2d(model2d, channel):
     bkg = np.polynomial.polynomial.polyval3d(r, d, l, c)
 
     # subtract background:
-    model2d.data[y, x] -= bkg
+    if subtract:
+        model2d.data[y, x] -= bkg
+        bkgmeta
```

Comment:
Is there supposed to be more to this line, like modifying or using the value of bkgmeta?

Reply:
@hbushouse You are right: this is a leftover from an earlier attempt to make this function set ``meta.background.subtracted`` - I gave up on this idea. Thank you for noticing this!